htdocs/install/upgrade2.php

```diff
@@ -1,10 +1,21 @@
 <?php
+
+/*
```

Comment:
i don't want to start this king of change and prefer to keep the header of file using the same structure everywhere so 
copyright, licence, comment. In 1 block.
Then doc for doxygen. This must remains in 1 block after. 
Some tools may rely on this and this does not bring a not enough enhancement to justify a hugh work (if we do for one file, we must do for all others and this is a very high work with to gain).

Reply:
okay agree